src/domains/shared-app/04_apim_statuspage.tf

```diff
@@ -31,7 +31,9 @@ locals {
     subscription_required = false
     service_url           = null
   }
-  aks_path = var.env == "prod" ? "weuprod.%s.internal.platform.pagopa.it" : "weu${var.env}.%s.internal.${var.env}.platform.pagopa.it"
+  aks_path           = var.env == "prod" ? "weuprod.%s.internal.platform.pagopa.it" : "weu${var.env}.%s.internal.${var.env}.platform.pagopa.it"
+  fe_backoffice_path = "pagopa${var.env_short}selfcaresa.z6.web.core.windows.net/ui/version.json"
```

Comment:
why not this https://github.com/pagopa/terraform-azurerm-v3/blob/a788d9cd14f2f1157b72aa0d011f75ad56b12b0b/storage_account/outputs.tf#L24 ? 
![image](https://github.com/pagopa/pagopa-infra/assets/36746022/c3aab86f-fd27-4dad-b9bb-1e273f664e72)

Reply:
Good idea! This will be changed as you describes, both for SelfCare BO Frontend and for ApiConfig Frontend URLs.